lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -128,19 +129,23 @@ def test_server_and_relay():
 
     @staticmethod
     @responses.activate
-    def test_relay():
+    @pytest.mark.parametrize("brief", (True, False))
+    def test_relay(brief):
         TestResultsPush.relay_mock()
         runner = CliRunner(mix_stderr=False)
-        result = runner.invoke(
-            main,
-            args=[
-                TestResultsPush.RELAY_SWITCH,
-                TestResultsPush.RELAY_TEXT,
-                tarball,
-            ],
-        )
+        arg_list = [
+            TestResultsPush.RELAY_SWITCH,
+            TestResultsPush.RELAY_TEXT,
+            tarball,
+        ]
+        if brief:
+            arg_list.append(TestResultsPush.BRIEF_SWITCH)
+        result = runner.invoke(main, args=arg_list)
         assert result.exit_code == 0
-        assert "RELAY log.tar.xz: http://relay.example.com/" in result.stdout
+        pattern = (
+            "RELAY log.tar.xz: " if not brief else ""
+        ) + r"http://relay.example.com/[a-z0-9]+\n"
+        assert re.match((pattern), result.stdout)
```

Comment:
What do the extra parens in the first argument to `match()` do?  (I'm surprised that Black left them and that Flake8 didn't object....)